### ISSUE TITLE
zeekctl print, netstats, and peerstats always segfault in broker.Endpoint()

### ISSUE THREAD

**Issue by bv3** [zeek/zeek#1737]
I've seen other reports of broker-related segfaults in zeekctl, but haven't come across this one.  We've only seen the issue with zeekctl print, netstats, and peerstats.

CentOS Linux release 8.4.2105

Installed rpms from https://download.opensuse.org/repositories/security:/zeek/CentOS_8/x86_64/
```
libbroker-lts-devel-4.0.3-1.4.x86_64
zeekctl-lts-4.0.3-1.4.x86_64
zeek-lts-4.0.3-1.4.x86_64
zeek-lts-btest-4.0.3-1.4.x86_64
zeek-lts-core-4.0.3-1.4.x86_64
zeek-lts-devel-4.0.3-1.4.x86_64
zeek-lts-libcaf-devel-4.0.3-1.4.x86_64
zeek-lts-zkg-4.0.3-1.4.x86_64

```
The backtraces for `print`, `netstats` and `peerstats` are almost identical, all terminating with a segfault at line 63 line in events.py, which is  `endpoint = broker.Endpoint().`
```
# python3 -q -X faulthandler /opt/zeek/bin/zeekctl print Log::enable_local_logging logger
Fatal Python error: Segmentation fault
. . .
Current thread 0x00007f12e0530b80 (most recent call first):
  File "/opt/zeek/lib/zeek/python/zeekctl/ZeekControl/events.py", line 63 in _send_event_init
  File "/opt/zeek/lib/zeek/python/zeekctl/ZeekControl/events.py", line 43 in send_events_parallel
  File "/opt/zeek/lib/zeek/python/zeekctl/ZeekControl/control.py", line 1113 in print_id
  File "/opt/zeek/lib/zeek/python/zeekctl/ZeekControl/zeekctl.py", line 470 in print_id
  File "/opt/zeek/lib/zeek/python/zeekctl/ZeekControl/zeekctl.py", line 40 in wrapper
  File "/opt/zeek/lib/zeek/python/zeekctl/ZeekControl/zeekctl.py", line 60 in wrapper
  File "/opt/zeek/bin/zeekctl", line 545 in do_print
  File "/usr/lib64/python3.6/cmd.py", line 217 in onecmd
  File "/opt/zeek/bin/zeekctl", line 790 in main
  File "/opt/zeek/bin/zeekctl", line 814 in <module>
Segmentation fault (core dumped)
```

Journalctl reports the following from systemd-coredump.
```
Process 2423308 (python3) of user 0 dumped core.

Stack trace of thread 279888:
#0  0x0000000000000000 n/a (n/a)

```

**Comment by 0xxon** [comment]
This one should definitely be fixed in all of our dot-releases - this is pretty obviously a regression :)

**Comment by JustinAzoff** [comment]
the zeekctl test suite is probably failing, but we haven't yet automated that as part of CI.

**Comment by timwoj** [comment]
This is possibly a dupe of https://github.com/zeek/zeek/pull/1734. Does it work correctly on 4.0.4?

**Comment by timwoj** [comment]
This appears fixed on 4.0.4, likely because the above linked PR. Closing this issue. Feel free to reopen it if it crops up again.
